core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -125,6 +125,20 @@ public ServerMain(String[] args) throws Exception {
                 break;
         }
         log.advisory().$("available CPUs: ").$(Runtime.getRuntime().availableProcessors()).$();
+        log.advisory().$("db root: ").$(configuration.getCairoConfiguration().getRoot()).$();
```

Comment:
shall we check backup path too?

Reply:
done here https://github.com/questdb/questdb/pull/1725/commits/444239783f1a60cd1886777a8eb03f12e9b458cf